### PR TITLE
TASK: Replacing codecov with coveralls

### PR DESCRIPTION
**Pull request by kevinfenger**

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -14,16 +14,16 @@ jobs:
...
           name: Run tests with coverage
           command: |
             source venv/bin/activate
             coverage run setup.py test
-            coverage xml -i
       - run:
-          name: Send coverage results to CodeCov
-          command: bash <(curl -s https://codecov.io/bash)
+          name: Send coverage results to Coveralls
+          command: coveralls
```
Comment (pachewise) [inline .circleci/config.yml]: I would pull from `master` to get a small fix I made. Second part, is, you need to source venv/bin/activate here to be able to run this

File: .circleci/config.yml [hunk 2]
```diff
@@ -14,16 +14,17 @@ jobs:
           command: |
             source venv/bin/activate
             pip install -r requirements.txt
-            pip install coverage
+            pip install coveralls
```
Comment (pachewise) [inline .circleci/config.yml:17]:
```suggestion
            pip install coverage coveralls
```

File: weighted_levenshtein/clev.pyx [hunk 3]
```diff
@@ -179,12 +179,12 @@ def damerau_levenshtein(
     if transpose_costs is None:
         transpose_costs = unit_matrix
 
-    s1 = str(str1).encode()  
-    s2 = str(str2).encode()  
+    # s1 = str(str1).encode()
+    # s2 = str(str2).encode()
```
Comment (pachewise) [inline weighted_levenshtein/clev.pyx:183]: @RevolutionTech @dsu1995 do any of you remember why we did this? It is breaking tests for this PR, but it also does not feel necessary (str1/str2 should already be unsigned char, so don't see an obvious reason to transform this before passing to `c_damerau_levenshtein`)

**Review by pachewise — COMMENTED** [review]
One question before we can merge this

File: weighted_levenshtein/clev.pyx (hunk 3 above)
Comment (RevolutionTech) [reply weighted_levenshtein/clev.pyx:183]: I remember this being related to Python 2/3 compatibility, but I don't remember much more than that.

It looks like duxan authored this change in https://github.com/duxan/weighted-levenshtein/commit/97428a59b5d9a07965792a323a2ae40d5bc8f634.

@duxan, do you remember?

**Pull request merged**